cpp/non_iid_main.cpp

```diff
@@ -435,27 +412,22 @@ int main(int argc, char* argv[]) {
         if (ret_min_entropy >= 0) {
             if (verbose == 1) printf("\tMulti Markov Model with Counting (MultiMMC) Prediction Test Estimate = %f / %d bit(s)\n", ret_min_entropy, data.word_size);
             H_original = min(ret_min_entropy, H_original);
+            tc639.ret_min_entropy = ret_min_entropy;
```

Comment:
Similarly, if both literal and bitstring assessments are conducted, only the literal assessment is saved.